tools/sggosemanticrelease/tools.go

```diff
@@ -23,7 +23,8 @@ func Command(ctx context.Context, args ...string) *exec.Cmd {
 	return sg.Command(ctx, sg.FromBinDir(name), args...)
 }
 
-func ReleaseFromCloudBuildCommand(ctx context.Context, ci bool, repo string) *exec.Cmd {
+// ReleaseFromCloudBuildCommand creates a semantic release with optional arguments to semantic-release.
+func ReleaseFromCloudBuildCommand(ctx context.Context, ci bool, repo string, argsOpt ...string) *exec.Cmd {
```

Comment:
Maybe call it `extraArgs`, one could otherwise argue that these should override rather than be appended I guess :shrug: 

```suggestion
func ReleaseFromCloudBuildCommand(ctx context.Context, ci bool, repo string, extraArgs ...string) *exec.Cmd {
```

Reply:
Good idea, fixed.